Source/santad/EventProviders/SNTDeviceManager.mm

```diff
@@ -199,43 +200,80 @@ - (void)handleAuthMount:(const es_message_t *)m
     return;
   }
 
-  long mountMode = m->event.mount.statfs->f_flags;
+  struct statfs *eventStatFS;
+  BOOL isRemount = NO;
+
+  switch (m->event_type) {
+    case  ES_EVENT_TYPE_AUTH_MOUNT:
+      eventStatFS = m->event.mount.statfs;
+      break;
+    case ES_EVENT_TYPE_AUTH_REMOUNT:
```

Comment:
Will the remount that santad initiates come through here? Can we avoid that, if so?

Reply:
The remount will come through because we're initiating it via diskarbitrationd. 

https://github.com/google/santa/blob/c7e309ccb149e0b7fba1979d73464a38735b516b/Source/santad/EventProviders/SNTDeviceManager.mm#L256-L257

We could prevent the remount message by doing the mount call in SNTDeviceManager and muting santad like in #820 .  That said we already have mocks for this.

Reply:
I'd say if we're going to redo the mounts let's do that in a future PR.